bigquery/tables.rb

```diff
@@ -213,10 +204,9 @@ def run_query_as_job project_id:, query_string:
   # project_id   = "your google cloud project id"
   # query_string = "query string to execute (using bigquery query syntax)"
 
-  require "google/cloud"
+  require "google/cloud/bigquery"
 
-  gcloud   = Google::Cloud.new project_id
-  bigquery = gcloud.bigquery
+  bigquery = Google::Cloud::Bigquery.new
```

Comment:
project: project_id is necessary because this is inside a function which has a project_id parameter.